examples/fluids/README.md

```diff
@@ -140,6 +140,22 @@ The following options are common among all problem types:
   - Use freestream boundary conditions on this list of faces
   -
 
+* - `-stats_enable`
+  - Enable collection of spanwise flow statistics
+  - `false`
+
+* - `-stats_test`
+  - Use manufactured solution for statistics collection (used for testing)
+  - `false`
```

Comment:
I don't think this is used in a test? If it's going to exist purely for development (and stay in to test accuracy if this gets futzed with), it'd be better to remove it from docs and from `-help` (by using `PetscOptionsGetBool()`).

Reply:
Yeah, it's not currently used as a test. And it doesn't make for a good regression test either. I'll move it out from help and the docs.